diff --git a/node_modules/@bacons/apple-targets/build/with-xcode-changes.js b/node_modules/@bacons/apple-targets/build/with-xcode-changes.js
index 96ded68..5d2b980 100644
--- a/node_modules/@bacons/apple-targets/build/with-xcode-changes.js
+++ b/node_modules/@bacons/apple-targets/build/with-xcode-changes.js
@@ -223,7 +223,7 @@ async function applyXcodeChanges(config, project, props) {
             config.getReferrers().forEach((ref) => {
                 ref.removeReference(config.uuid);
             });
-            config.removeFromProject();
+            config?.removeFromProject?.();
         });
         // Remove existing build configuration list
         targetToUpdate.props.buildConfigurationList
@@ -231,7 +231,7 @@ async function applyXcodeChanges(config, project, props) {
             .forEach((ref) => {
             ref.removeReference(targetToUpdate.props.buildConfigurationList.uuid);
         });
-        targetToUpdate.props.buildConfigurationList.removeFromProject();
+        targetToUpdate.props.buildConfigurationList?.removeFromProject?.();
         // Create new build phases
         targetToUpdate.props.buildConfigurationList =
             (0, configuration_list_1.createConfigurationListForType)(project, props);
@@ -383,7 +383,7 @@ async function applyXcodeChanges(config, project, props) {
     }
     else {
         // Remove the exception set if there are no shared assets.
-        existingExceptionSet === null || existingExceptionSet === void 0 ? void 0 : existingExceptionSet.removeFromProject();
+        existingExceptionSet === null || existingExceptionSet === void 0 ? void 0 : existingExceptionSet?.removeFromProject?.();
     }
     function configureTargetWithGlobalSharedAssets(target) {
         var _a;
